recipes/application.rb

```diff
@@ -46,10 +48,10 @@
 # install django
 application app_name do
   path node[:cyclesafe_chef][:directory]
-  owner node[:cyclesafe_chef][:user]
+  owner node[:cyclesafe_chef][:user] 
   group node[:cyclesafe_chef][:group]
   repository node[:cyclesafe_chef][:repository]
-  revision node[:cyclesafe_chef][:revision]
+  revision node[:cyclesafe_chef][:revision] if node[:cyclesafe_chef][:revision]
```

Comment:
revision has a default value to just pull from the HEAD of the deploy branch. This allows it to do this if revision is not defined.